### PR TITLE
chore: bring back prebuilt binaries

### PR DESCRIPTION
**Pull request by ralphtheninja**
Closes https://github.com/deltachat/deltachat-node/issues/320

### CODE REVIEW

**Comment by ralphtheninja** [comment]
Going to try pushing a temporary tag later to try it out.

File: README.md [hunk 1]
```diff
@@ -955,10 +955,8 @@ We have the following scripts for building, testing and coverage:
 - `npm run coverage-html-report` Generates a html report from the coverage data and opens it in a browser on the local machine.
 - `npm run generate-constants` Generates `constants.js` and `events.js` based on the `deltachat-core-rust/deltachat-ffi/deltachat.h` header file.
 - `npm install` After dependencies are installed, runs `node-gyp-build` to see if the native code needs to be rebuilt.
-- `npm run rebuild-all` Rebuilds all the code.
-- `npm run rebuild-core` Builds the `deltachat-core-rust` submodule.
-- `npm run rebuild-bindings` Builds `deltachat.node`.
-- `npm run postinstall` Copies `deltachat.dll` from `deltachat-core-rust` to `build/Release`. Windows only.
+- `npm run node-gyp-build` Tries to load prebuilts and falls back to rebuilding the code.
+- `npm run prebuild` Builds prebuilt binary to `prebuilds/$PLATFORM-$ARCH`. Copies `deltachat.dll` from `deltachat-core-rust` for windows.
```
Comment (Jikstra) [inline README.md]: I never understood the prebuild command. Why does it rebuild the core and does prebuild stuff? I prefer having  rebuild-* commands and a prebuildify command to make a prebuild out of the current build. I sometimes only want to rebuild the bindings or the core or do the prebuilds.

Comment (ralphtheninja) [reply README.md]: I can put back some commands to enable you to rebuild each part, but I'm keeping `prebuildify` and `node-gyp-build`.

File: package.json [hunk 2]
```diff
@@ -7,9 +7,9 @@
     "coverage-html-report": "rm -rf coverage/ && nyc report --reporter=html && opn coverage/index.html",
     "generate-constants": "./scripts/generate-constants.js",
     "install": "node scripts/install.js",
```
Comment (ralphtheninja) [inline package.json]: Normally you would do the `node-gyp-build` command in the `install` script, but we have another seam there due to the system core.

File: scripts/postinstall.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+      'build',
+      'Release',
+      'deltachat.dll'
+    )
+  }
+}
+
+const dest = getDestination()
+
+copy(from, dest, (err) => {
```
Comment (Jikstra) [inline scripts/postinstall.js]: Why is copying only needed on windows?

Comment (ralphtheninja) [reply scripts/postinstall.js]: Because we need `deltachat.dll` from `deltachat-core-rust`.

Comment (Jikstra) [reply scripts/postinstall.js]: But isn't this the same on linux/mac, only difference is that we need a .so/.dylib?

**Comment by ralphtheninja** [comment]
Hold off with merge. I want to publish a test release first, wipe the version back to what it was and then squash.

File: scripts/postinstall.js (hunk 3 above)
Comment (ralphtheninja) [reply scripts/postinstall.js]: No since we're linking statically on linux and mac.

Comment (Jikstra) [reply scripts/postinstall.js]: Oh okay thanks :)

**Comment by Jikstra** [comment]
Looks very good to me, merge it yourself whenever you want :)

**Pull request merged**